crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -1050,8 +1050,8 @@ pub struct DevInspectArgs {
     pub gas_objects: Option<Vec<ObjectRef>>,
     /// Whether to skip transaction checks for the transaction.
     pub skip_checks: Option<bool>,
-    /// Whether to return the raw transaction data.
-    pub show_raw_txn_data: Option<bool>,
+    /// Whether to return the raw transaction data and effects.
+    pub show_raw_txn_data_and_effects: Option<bool>,
```

Comment:
Is there any compatibility issue with this change?

Reply:
Should be backwards compatible since it's an `Option`

Reply:
I guess the question here is whether there's an issue for people who were using this flag before just to get the raw tx data and now they get both data and effects, but I'm hoping/guessing that you added this flag recently and so we (GraphQL) are the only customer?

Reply:
Correct. This is only two days old.